src/setfit/trainer.py

```diff
@@ -3,6 +3,7 @@
 
 import evaluate
 import numpy as np
+import torch
```

Comment:
Hi @tomaarsen, maybe you could place the `torch` import inside the `TYPE_CHECKING` block as described my following comment